proto/frequenz/api/microgrid/v1/microgrid.proto

```diff
@@ -325,10 +304,10 @@ service Microgrid {
   //
   // If any of the above mentioned actions for a given component has already
   // been performed, then this method call efffectively skips that action.
-  rpc ColdStandbyComponent(ColdStandbyComponentRequest)
+  rpc StandbyComponent(StandbyComponentRequest)
```

Comment:
This sounds strange. We are putting a component into standby. So maybe something `EnableComponentStandby` or something similiar? Here are some suggestions: 

* ToggleComponentStandbyMode
* SetComponentToStandby
* EnableStandbyForComponent
* SwitchToStandbyMode
* PutComponentInStandby
* SetStandbyState
* ActivateStandbyMode / DeactivateStandbyMode
* ToggleStandbyState
* ChangeToStandbyMode

My question would be if a user puts an inverter into standby how can he activate it again? I think this we might also want to make clear in the documentation.

Reply:
My preference would be to use a verb at the front e.g. `SetComponentToStandby` or `ActivateStandbyMode` would be my favourites.

Reply:
`PutComponentInStandby` sounds good. I have updated the new RPC name to `PutComponentInStandby`.

>My question would be if a user puts an inverter into standby how can he activate it again? I think this we might also want to make clear in the documentation.

Good call. I added it to the RPC docs.

Reply:
`StandbyComponent` also sounded a bit weird to me at first, but I think it had a nicer symmetry with `StartComponent` and `StopComponent`, IMHO now it is harder to see those 3 methods are related. As a final suggestion, I would put `PauseComponent` to the table, but I don't have any major complains if you prefer to keep `PutComponentInStandby`.

Reply:
(just unresolved so my last comment is not lost)

Also, not sure if it would be worth clarifying in `SetComponentPowerActive` docs that this will start the component if it was in standby/paused.